data/morphology/grab_unimorph_data.py

```diff
@@ -6,30 +6,32 @@
 import os
 import time
 
-from typing import Dict
+from typing import Dict, List
 
 import requests
 
 
 UNIMORPH_DICT_PATH = "unimorph_languages.json"
 
 
-def download(data_to_grab: Dict[str, str]) -> Dict[str, str]:
+def download(data_to_grab: Dict[str, List[str]]) -> Dict[str, List[str]]:
     to_retry = {}
     os.mkdir("tsv")
-    for language, url in data_to_grab.items():
-        with requests.get(url, stream=True) as response:
-            logging.info("Downloading: %s", language)
-            if response.status_code == 200:
-                with open(f"tsv/{language}.tsv", "wb") as f:
-                    f.write(response.content)
-            else:
-                logging.info(
-                    "Status code %d while downloading %s",
-                    response.status_code,
-                    language,
-                )
-                to_retry[language] = data_to_grab[language]
+    for language, urls in data_to_grab.items():
+        with open(f"tsv/{language}.tsv", "wb") as sink:
+            for url in urls:
+                with requests.get(url, stream=True) as response:
```

Comment:
I think this might borrow too much from the `grab_wortschatz_data.py` script. All of the checking of status codes and `stream=True`, `to_retry` and `time.sleep` stuff was because downloading the Wortschatz data often failed in exotic ways. I doubt we need any of it here, but better safe than sorry?

Reply:
I see why you say that but FWIW a few of these files are on the order of 100 MB and it's easy to imagine it hanging up with something that big. I figured since it ain't broke...